### PR TITLE
Feature/ui sign up

### PR DESCRIPTION
**Pull request by pkvarnwal**

### CODE REVIEW

File: app/app.iml [hunk 1]
```diff
@@ -83,6 +83,7 @@
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/incremental-safeguard" />
```
Comment (ashutoshsingh33) [inline app/app.iml]: add this file in git ignore

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -21,4 +21,49 @@
...
     </style>
+
+    <style name="ToolbarTitleAppearance" parent="@style/TextAppearance.Widget.AppCompat.Toolbar.Title">
+        <item name="android:textSize">@dimen/_16ssp</item>
+        <item name="android:textAllCaps">false</item>
+        <item name="android:layout_gravity">center</item>
+        <item name="android:gravity">center</item>
+        <item name="android:textColor">@color/eqdepot_blue</item>
+    </style>
+
```
Comment (ashutoshsingh33) [inline app/src/main/res/values/styles.xml]: Please remove the extra space.

**Comment by ashutoshsingh33** [comment]
@aashishdhawan please review.

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -25,5 +25,21 @@
             android:name=".activities.MainActivity"
             android:screenOrientation="portrait"
             android:theme="@style/showNotificationBar" />
+
```
Comment (aashishdhawan) [inline app/src/main/AndroidManifest.xml]: Make sure complete app is portrait and not every single activity is done this way.

File: app/src/main/java/com/eqdepot/activities/LoginActivity.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package com.eqdepot.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
```
Comment (aashishdhawan) [inline app/src/main/java/com/eqdepot/activities/LoginActivity.java]: why there is extra lines?

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,3 +1,21 @@
...
+    <string name="register_with_us">Register With Us</string>
+    <string name="catalog">Catalog</string>
+    <string name="first_name">First Name</string>
+    <string name="last_name">Last Name</string>
+    <string name="email_address">Email Address</string>
+    <string name="phone_number">Phone Number</string>
+    <string name="read_agree">I have read &amp; agree all the</string>
+    <string name="term_condition"> terms &amp; condition</string>
+    <string name="register">Register</string>
+    <string name="number">123453132</string>
```
Comment (aashishdhawan) [inline app/src/main/res/values/strings.xml]: Please add a TODO here and every time you do something like that

**Pull request merged**